BUILD.md

```diff
@@ -0,0 +1,50 @@
+# Building the Project
+
+The following steps should get your C SDK for Ziti building. C development is specific to your operating system and 
+tool chain used. These steps should work properly for you but if your OS has variations you may need to adapt these steps accordingly.
+
+## Prerequisites
+
+This repository expects the user to have at least a basic understanding of what a Ziti Network
+is. To use this library it is also required to have a functioning Ziti Network availalbe to use.
+To learn more about what Ziti is or how to learn how to setup a Ziti Network head over to [the official documentation
+site](https://openziti.github.io/ziti/overview.html).
+
+## Building Requirements
+
+* [Cmake (3.22+)](https://cmake.org/install/)
```

Comment:
not that it really matters, but the current cmake is 3.26.3.

Reply:
I took what was in CMakeLists.txt. I also considered removing the version, since the last time it was updated was 3.12 :/